# Wing deployment command message
uint64 timestamp # time since system start (microseconds)
bool deploy      # true to deploy wings, false to retract
